emukit/bayesian_optimization/acquisitions/probability_of_improvement.py

```diff
@@ -67,3 +67,56 @@ def evaluate_with_gradients(self, x: np.ndarray) -> Tuple:
     @property
     def has_gradients(self):
         return isinstance(self.model, IDifferentiable)
+    
+    
+    
+class ProbabilityOfFeasibility(Acquisition):
```

Comment:
Could you add a test for this please? The tests are already written you just need to add a few things in `test_acquisitions.py`:
1. Import this acquisition.
2. A fixture that creates this acquisition
```
@pytest.fixture
def probability_of_feasibility_acquisition(gpy_model):
    return ProbabilityOfFeasibility(gpy_model)
```
3. Add this to the `acquisition_tests` list in the same file:
`acquisition_test_tuple('probability_of_feasibility_acquisition', True, default_grad_tol)`

Reply:
Hi, I have updated test_acquisitions.py following the instructions